BUILDING_ON_LINUX.md

```diff
@@ -34,50 +35,48 @@ _most likely works the same for other Debian-like distros_
 
 ### Manually
 
-1. `sudo pacman -S qt5-base qt5-multimedia qt5-svg qt5-tools gst-plugins-ugly gst-plugins-good boost rapidjson pkgconf openssl cmake`
-1. go into project directory
-1. create build folder `mkdir build && cd build`
+1. Install all of the dependencies using `sudo pacman -S qt5-base qt5-multimedia qt5-svg qt5-tools gst-plugins-ugly gst-plugins-good boost rapidjson pkgconf openssl cmake`
+1. Go into the project directory
+1. Create a build folder and open it (`mkdir build && cd build`)
+1. Use one of the options below to compile it
 
-#### Using QMake
+### Using CMake
 
-1. `qmake .. && make`
+`cmake .. && make`
 
-#### Using CMake
+### Using QMake
 
-1. `cmake .. && make`
+`qmake .. && make`
```

Comment:
Why was the order reversed?

Reply:
> -CMake is now always in front of QMake, should make users use CMake instead of QMake, since QMake is going to be deprecated soon. (Excluding the FreeBSD guide)